src/TodoByPackageVersionRule.php

```diff
@@ -58,6 +58,12 @@ public function __construct(
     ) {
         $this->workingDirectory = $workingDirectory;
         $this->errorBuilder = $errorBuilder;
+
+        // require the top level installed versions, so we don't mix it up with the one in phpstan.phar
+        $installedVersions = $this->workingDirectory . '/vendor/composer/InstalledVersions.php';
```

Comment:
One can configure an alternative folder instead of `vendor/`: https://getcomposer.org/doc/06-config.md#vendor-dir
So hard coding might fail for some users.

Reply:
I am aware this config exists, but AFAICS there are a lot of tools which will break when using different vendor-dir.

Reply:
Not until now, I've opened an separate Issue: https://github.com/staabm/phpstan-todo-by/issues/85